scripts/build_ros1_bridge.bash

```diff
@@ -143,7 +143,9 @@ if [ -z $ROS1_WS_DIR ]; then
   echo "ROS1 workspace does not exist!"
   exit 1
 else
-  if [ -f $ROS1_WS_DIR/install/setup.bash ]; then
+  if [ -f $ROS1_WS_DIR/devel/setup.bash ]; then
```

Comment:
Can you rather have a check for which dir exists? Because ROS1 workspaces can also be built with colcon and this goes to install

Reply:
@TSC21 If devel does not exist, it tries to source install/setup.bash. See the next lines. In `build_all.sh` this is handled exactly the same way

Reply:
You are right, sorry! Thanks!